easybuild/easyconfigs/c/carma/carma-2.01-foss-2019b.eb

```
@@ -0,0 +1,34 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Helena Gomez <helena.gomez@hpcnow.com>, Erica Bianco, Danilo Gonzalez, Jordi Blasco
+# License::     GPL-v3.0
+# #
+
+easyblock = 'Tarball'
+
+name = 'carma'
```

According to the homepage, the name should be capitalized:
```suggestion
name = 'Carma'
```